app/components/service/serviceController.js

```diff
@@ -37,6 +37,8 @@ function ($q, $scope, $stateParams, $state, $location, $anchorScroll, ServiceSer
   };
 
   $scope.goToItem = function(hash) {
+    $state.params.page_anchor = hash;
+    $location.path('/service/'+$scope.service.Id+'/'+hash);
```

Comment:
Instead of using this hack and `$location.path` could we use `$location.hash` and `$anchorScroll` ? 

Something like: 

```
$scope.goToItem = function(hash) {
  var newHash = hash;
  if ($location.hash() !== newHash) {
    $location.hash(hash);
  } else {
    $anchorScroll();
  }
};
```

As presented in the second example here: https://docs.angularjs.org/api/ng/service/$anchorScroll